### PR TITLE
[198] Web create edit freak modal

### PR DESCRIPTION
**Pull request by MarianBadea**
This PR will create the edit freak modal

### CODE REVIEW

**Comment by MarianBadea** [comment]
[Web: Create Edit Freak Modal](https://trello.com/c/rrDd10sD/198-web-create-edit-freak-modal)

**Review by adrianbunea — CHANGES_REQUESTED** [review]
Just one comment for this

**Review by adrianbunea — APPROVED** [review]

**Review by luism-s — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 1]
```diff
@@ -15,15 +22,33 @@ function App() {
...
+          <Link to="/freaks">
+            <button type="button" className="app__agile">
+              <img className="app__agile-img" src={ logo } alt="Nav" />
+              <p>Freaks</p>
+            </button>
+          </Link>
+          <Link to="/">
+            <button type="button" className="app__agile">
+              <FontAwesomeIcon icon="tasks" />
+              <p>Projects</p>
```
Comment (adrianbunea) [inline src/App.js:49]: BEM - Block, Element, Modifier, explain how "agile" is an `element` of the "app" `block`.

File: src/components/ConfirmDeleteModal/ConfirmDeleteModal.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <Modal
+      className="af-modal__fit-to-content"
+      title={ title }
+      isOpen={ isOpen }
+      footerContent={ getFooter() }
+      onClose={ onClose }
+    >
+      <div className="delete-message">
+        <h1>Warning!</h1>
+        <p>Are you sure you want to delete this freak?</p>
```
[the remark this answers is not in the document]
Comment (adrianbunea) [reply src/components/ConfirmDeleteModal/ConfirmDeleteModal.jsx:39]: @MarianBadea bump

**Review by adrianbunea — CHANGES_REQUESTED** [review]

**Pull request merged**